barman/infofile.py

```diff
@@ -878,7 +898,7 @@ def get_child_backup_info(self, child_backup_id):
 
         return None
 
-    def walk_backups_tree(self):
+    def walk_backups_tree(self, return_root=True):
```

Comment:
I ended up touching this function! Would it be ok to make the modification in this PR?

Reply:
I think so -- we are going to use it here after all, and the original ticket used to create this method has already been merged.

Reply:
yeah seems good to me doing it here